test/integration/render-tests/globe/globe-transition/ocean-artifact/style.json

```diff
@@ -4,7 +4,7 @@
     "test": {
       "width": 512,
       "height": 512,
-      "allowed": 0.0003
+      "allowed": 0.013
```

Comment:
This is a very high allowed, but I think it's ok for this test since it's purpose was to catch a bug with a large area of missing screen.